tests/integration/components/stepper-signup-test.js

```diff
@@ -0,0 +1,17 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'website-www/tests/helpers';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | stepper-signup', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('stepper-signup page render', async function (assert) {
+    // Set any properties with this.set('myProperty', 'value');
+    // Handle any actions with this.set('myAction', function(val) { ... });
+    assert.expect(1);
+    await render(hbs`<StepperSignup />`);
+
+    assert.dom('h1').hasText('Hello world');
```

Comment:
Use data-test selector here and also why are we testing hello world here, add the text which actually should exist here.

Reply:
Done